file_mapper_script.py

```diff
@@ -136,6 +164,8 @@ def parse_data(contents, verbose=False, testdebug=False):
             if verbose:
                 print("Source already exists in json data")
             source = os.path.join(data['SOURCE'],key)
+        else:
+            source = key
```

Comment:
Nice catch.  So would this imply you are doing file-mapping with an absolute path in your source keys instead of just a relative one?  The original concept was to (basically always) provide a source argument either in the file-mapper JSON or in the file_mapper_script.py call.  This is an interesting thought.  Perhaps print the user a warning they did not provide a source argument anywhere and it is defaulting to `key`?

Reply:
For my initial testing I had simply used the full path names for both the SOURCE and DEST keys in the json file. Moving forward I will probably also move to the intended approach, as you mentioned, as it is more flexible. 

0b95714 adds a printout to warn the user